templates/components/itilobject/timeline/form_document_item.html.twig

```diff
@@ -88,6 +88,20 @@
                      </button>
                   </form>
                {% endif %}
+                {% if entry_i['_can_edit'] %}
+                    {% set blacklisted_class    = document['item']['is_blacklisted'] ? 'red' : '' %}
+                    {% set blacklisted_title    = entry_i['is_blacklisted'] ? _x('button', 'Remove from import exclusion list') : _x('button', 'Add to import exclusion list') %}
+                     <form class="d-inline" method="post" action="{{ call('Document::getFormURL') }}">
+                        <input type="hidden" name="id" value="{{ document['item']['id'] }}">
+                        <input type="hidden" name="is_blacklisted" value="{{ document['item']['is_blacklisted']|reverse }}">
```

Comment:
The `reverse` filter is used to reverse strings and arrays, not toggle boolean: https://twig.symfony.com/doc/3.x/filters/reverse.html.

Reply:
fixed